features/test_feature.py

```diff
@@ -61,7 +61,8 @@
               'thrust_swap_ranges', 'thrust_uninitialized_fill_n', 'thrust_equal', 'system_atomic', 'thrust_detail_types',
               'operator_eq', 'operator_neq', 'operator_lege', 'thrust_system', 'thrust_reverse_copy',
               'thrust_device_new_delete', 'thrust_temporary_buffer', 'thrust_malloc_free', 'codepin', 'thrust_unique_count',
-              'thrust_advance_trans_op_itr', 'cuda_stream_query', "matmul", "transform"]
+              'thrust_advance_trans_op_itr', 'cuda_stream_query', "matmul", "transform",
+              "graphics_texture"]
```

Comment:
1st use of this name?

Reply:
Pushed the commit to correct the test API name